### PR TITLE
adding log_level parameter - fixes #8939

### PR DESCRIPTION
**Pull request by gpetras**

### CODE REVIEW

**Comment by mmoll** [comment]
Please add documentation for the parameter in the header of init.pp, kafo/foreman-installer needs this.
Looking fine otherwise :+1:

**Comment by mmoll** [comment]
also, the commit message needs to reference the correct issue ;

**Comment by ekohl** [comment]
What @mmoll said.

**Comment by gpetras** [comment]
lol, thanks. I'll get that fixed.

File: manifests/init.pp [hunk 1]
```diff
@@ -24,6 +24,8 @@
 #
 # $log::                        Foreman proxy log file
 #
+# $log_level::                  Foreman proxy log level, ie INFO,i DEBUG, FATAL, etc.
```
Comment (mmoll) [inline manifests/init.pp]: typo here, will squash a fix into the merge

**Comment by mmoll** [comment]
merged as 10ee4ce, thanks @gpetras :tada:

**Pull request closed without merging**